src/client/swagger/unity.yml

```diff
@@ -23,174 +23,197 @@ paths:
+            enum: [limits, billing, storage_gb, write_mb, reads_gb, query_count]
+          description: The name of the UsageVector to query.
+          example: 'reads_gb'
+        - in: query
+          name: start
+          schema:
+            type: string
+            enum: [24h, 7d, 30d]
+            default: 24h
       responses:
         '200':
-          description: billing notification settings updated
+          description: Usage history for account
           content:
-            application/json:
+            application/csv:
               schema:
-                $ref: '#/components/schemas/BillingNotifySettings'
+                type: string
+                example: |
+                  #group,false,false,true,false,false
+                  #datatype,string,long,string,dateTime:RFC3339,long
+                  #default,reads_gb,,,,
+                  ,result,table,_field,_time,_value
+                  ,,0,reads_gb,2021-01-13T22:05:00Z,120
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/contact':
+  '/settings/notifications':
     get:
-      operationId: GetBillingContact
+      operationId: GetSettingsNotifications
       tags:
-        - BillingContact
+        - Settings
       responses:
         '200':
-          description: A billing contact object
+          description: Settings for billing notifications
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/BillingContact'
+                $ref: '#/components/schemas/BillingNotifySettings'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
     put:
-      operationId: PutBillingContact
+      operationId: PutSettingsNotifications
       tags:
-        - BillingContact
+        - Settings
       requestBody:
-        description: BillingContact to update
+        description: Update to notify setting
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/BillingContact'
+              $ref: '#/components/schemas/BillingNotifySettings'
       responses:
         '200':
-          description: billing contact updated
+          description: billing notification settings updated
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/BillingContact'
+                $ref: '#/components/schemas/BillingNotifySettings'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/cc':
-    get:
-      operationId: GetCCPageParams
+  '/billing/contact':
```

Comment:
Quick question: why is the billing contact a separate API query here? Wouldn't that be considered part of the [billing information](https://github.com/influxdata/ui/pull/666/files#diff-888a46184afed8040a74345087d09db51553252713b486dbd57610e728c89d02R33)?

Reply:
Billing contact is a separate entity in Quartz. It is added on the checkout page, but can be viewed and edited on the billing page.

Reply:
Gotcha. Thanks!

Reply:
This makes me think that there also needs to be a get for /billing/contact for displaying the existing billing contact on the billingn page. Rather than just updating it.

Reply:
Oh just kidding, i see thats a part of just `/billing`. That seems weird to me, but ill leave it for now.